app/models/concerns/pxe_loader_suggestion.rb

```diff
@@ -1,7 +1,11 @@
 module PxeLoaderSuggestion
   extend ActiveSupport::Concern
   def suggest_default_pxe_loader
-    self.pxe_loader ||= try(:operatingsystem).try(:parameters).try(:find_by_name, "pxe-loader").try(:value)
-    self.pxe_loader ||= try(:operatingsystem).try(:preferred_loader) if respond_to?(:pxe_loader)
```

Comment:
I'm trying to understand the logic here. Looking at the code the `respond_to?(:pxe_loader)` bit was already redundant since both classes that include this concern (host and hostgroup) have an explicit pxe_loader field.